src/overlays/actors/ovl_Bg_Spot09_Obj/z_bg_spot09_obj.c

```diff
@@ -21,25 +24,165 @@ const ActorInit Bg_Spot09_Obj_InitVars = {
     (ActorFunc)BgSpot09Obj_Update,
     (ActorFunc)BgSpot09Obj_Draw,
 };
-*/
-#pragma GLOBAL_ASM("asm/non_matchings/overlays/actors/ovl_Bg_Spot09_Obj/func_808B1AE0.s")
 
-#pragma GLOBAL_ASM("asm/non_matchings/overlays/actors/ovl_Bg_Spot09_Obj/func_808B1BA0.s")
+static u32 D_808B1F90[] = { 0x00000000, 0x06005520, 0x0600283C, 0x06008458, 0x06007580 };
```

Comment:
trailing comma (I think we're doing that even for single line?)

Reply:
Actually, the trailing comma should be omitted if you want clang-format to keep it in a single line.

Reply:
alright. the single line one below has a comma so I wasn't sure

Reply:
By a single line I mean with braces in the same line, which the one below doesn't have.